### PR TITLE
add media queries for video sizing

### PR DESCRIPTION
**Pull request by samhstn**
See #331

### CODE REVIEW

File: web/static/css/app.css [hunk 1]
```diff
@@ -512,3 +512,61 @@ code {
...
+}
+
+@media screen and (min-width: 60em) and (max-width: 1039px) {
+  .homepage_video_container {
+    width: 350px;
+    height: 196px;
+  }
+}
+
+@media screen and (max-width: 60em) {
```
Comment (iteles) [inline web/static/css/app.css]: Is there a reason we're mixing `em`s and `px`s in the media queries for this?

**Review by iteles — APPROVED** [review]
LGTM. As per my comment in the issue, I think we might have to end up making the video stack a little earlier so that we don't have a tiny video and mega padding to the right, but not a high priority right now 👍 

screen shot 2017-06-27 at 14 19 18

File: web/static/css/app.css (hunk 1 above)
Comment (shouston3) [inline web/static/css/app.css]: I used pixels to get the exact screen widths given to me by chrome when resizing the screen and the black lines on video appear

`em`s were used to match the exact break points for tachyon media query implementations

**Review by SimonLab — APPROVED** [review]
LGTM, thanks for the review @iteles :+1:

**Pull request merged**